lib/experimental/fonts-library/class-wp-font-family.php

```diff
@@ -0,0 +1,416 @@
+<?php
+/**
+ * Fonts Library Family class.
+ *
+ * This file contains functions and definitions for the Gutenberg Fonts Library.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
```

Comment:
To avoid fatal error when the class is introduced into Core, add a guard to the opening of the file. This guard will not be merged into Core, but rather will stay in Gutenberg.

```suggestion
if ( class_exists( 'WP_Font_Family' ) {
	return;
}
```

This approach is consistent with other `WP_` classes within Gutenberg, e.g. https://github.com/WordPress/gutenberg/blob/trunk/lib/experimental/class-wp-rest-block-editor-settings-controller.php#L9.

Reply:
With the merge of https://github.com/WordPress/gutenberg/pull/52696, CI should (🤞) automatically flag such unguarded classes (and functions)